docs/integrations/golang.md

```diff
@@ -81,7 +81,7 @@ Predefined build tags:
 #### `strategy: <string>`
 
 Manually specify the golang package manager being used. This should be untouched unless fossa is incorrectly attempting to analyze the wrong strategy. If this option is set, it is recommended [lockfile](#LockfilePath:-<string>) and [manifest](#ManifestPath:-<string>) be set as well. A list of supported strategies is as follows:
-- ```gomodules, dep, gdm, glide, godep, govendorvndr, gopath-vcs```
+- ```manifest:gomodules, manifest:dep, manifest:gdm, manifest:glide, manifest:godep, manifest:govendor, manifest:vndr, manifest:gopath-vcs```
```

Comment:
For the last one, `manifest:gopath-vcs`, [this line](https://github.com/fossas/fossa-cli/blob/7ad3119edbb1c85175a8c1542f40c45be9cf8d5e/analyzers/golang/analyze.go#L95) suggests it should simply by `gopath-vcs`. Or am I misunderstanding something?